lib/javascript.d.ts

```diff
@@ -334,8 +334,8 @@ declare global {
 
 			/** Configures this state as an alias for another state */
 			alias?: {
-				/** The target state id */
-				id: string;
+				/** The target state id or two target states used for reading and writing values */
+				id: string | { read?: string; write?: string };
```

Comment:
Having both properties optional would allow an alias with `id: {}`, which is incorrect.
I think it's fair to require both read and write if the object variant is used.

Reply:
I'll ask

Reply:
If only one of the properties is needed, we could use
```
AtLeastOne<{ read: string; write: string }>
```
to enforce that no empty object is given.

Reply:
I tested changing one of my aliases to either `read`+`write`, `read`-only or `write`-only. Only `read`+`write` works using the UI, the others fail with `Cannot write object: Alias id is invalid: The id is empty! Please provide a valid id.`.

Reply:
OK, both are required: https://github.com/ioBroker/ioBroker.js-controller/blob/HEAD/packages/adapter/lib/adapter/adapter.js#L2336-L2343